scripts/create-instance.sh

```diff
@@ -23,77 +23,77 @@
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
 # check environment variables:
-if [ -v ${INVENIO_WEB_HOST} ]; then
+if [[ -v INVENIO_WEB_HOST ]]; then
```

Comment:
`[[` is nice, but non-POSIX.  (If we ever have to target non-bash scenarios.) An alternative fix would be to use quotes that we use elsewhere, e.g. https://github.com/inveniosoftware/invenio/blob/maint-1.2/scripts/create-instance.sh#L29